lib/yelp/fusion/singleton.rb

```diff
@@ -0,0 +1,43 @@
+module Yelp
+  module Fusion
+    module Singleton
+      # These methods were copied from the old yelp implementation
```

Comment:
@lkatzman Remove this. These methods rely on completely new code. We don't need to indicate every place our gem bears resemblance to the old gem.